### PR TITLE
[WIP] Add draft post introducing distopia

### LINKED ISSUE #265: Distopia blog post
I am planning on making a blog post about distopia that we can possibly also make a JOSS paper. 

I have it about half written and will hopefully have it done around the same time as the distopia code will be ready to be merged.

### PR DESCRIPTION
**Pull request by hmacdope**
Fixes #265.

Still quite WIP, missing figures and inline math.
Would love some inital feedback and possibly help @richardjgowers.

### CODE REVIEW

File: _posts/2023-03-03-distopia.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
+---
+layout: post
+title: Introducing Distopia
+---
+
+Calculating Euclidian distances under periodic boundary conditions is a core component in both running and analysing molecular simulations. For example, the calculation of radial distribution functions, hydrogen bonds, structure factors, and many other common analyses require calculation of one or more pairwise Euclidian distances.
+
+The Euclidian distance formula under periodic boundary conditions can be expressed as:
```
Comment (orbeckst) [inline _posts/2023-03-03-distopia.md:8]: define R and A

File: _posts/2023-03-03-distopia.md [hunk 2]
```diff
@@ -0,0 +1,358 @@
+---
+layout: post
+title: Introducing Distopia
+---
+
+Calculating Euclidian distances under periodic boundary conditions is a core component in both running and analysing molecular simulations. For example, the calculation of radial distribution functions, hydrogen bonds, structure factors, and many other common analyses require calculation of one or more pairwise Euclidian distances.
```
Comment (orbeckst) [inline _posts/2023-03-03-distopia.md:6]: perhaps link to a wiki page on PBC or cite a paper ... making it more scholarly

comment on different "simulation boxes" would be helpful

Image illustrating PBC (at least in 2D) would be icing on the cake.

File: _posts/2023-03-03-distopia.md [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+    r.store(output);
+```
+
+Notice that the subtraction, multiplication and addition operations are all applied to each dimension of the coordinates. The sqrt and store operations are also applied to the entire SIMD register, resulting in 
+fantastic performance. 
+
+
+## Conclusion
+
+In this post we have seen how we can use the `distopia` library to accelerate distance calculations. Distopia is available as a backend to some MDAnalysis distiance functions right now, with support for more to be added in the future.
```
Comment (orbeckst) [inline _posts/2023-03-03-distopia.md:329]: distance

File: _posts/2023-03-03-distopia.md [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+
+The Euclidian distance formula under periodic boundary conditions can be expressed as:
+
+$$
+R = \sqrt{A}
+$$
+
+Many molecular mechanics engines have highly optimised code for calculating distances that often make use of specialised code paths and/or accelerators to achieve maximal performance. However this functionality is often heavily "baked-in" and is not able to be used as a standalone component.
+
+We at MDAnalysis wanted to bring some of this blazing speed to our distance calculations without sacrificing the modularity of MDAnalysis by introducing heavyweight dependencies on a simulation engine.
```
Comment (orbeckst) [inline _posts/2023-03-03-distopia.md:16]: Say early on how to use it in MDA and from which release onwards it is available. 

The detail below is really informative and it's great to have technical blog posts. I would just make sure that our general users get the message that (1) we have fast distance calculations and (2) you can use them right away.

**Review by orbeckst — COMMENTED** [review]
I look forward to seeing the post!!

**Comment by orbeckst** [comment]
@hmacdope does this announcement need to be completely rewritten, given the switch to highway?

**Comment by hmacdope** [comment]
@orbeckst correct

**Pull request closed without merging**